anchore_engine/common/helpers.py

```diff
@@ -29,19 +29,24 @@ def make_response_error(errmsg, in_httpcode=None, details=None):
 
     if isinstance(errmsg, Exception):
         if 'anchore_error_json' in errmsg.__dict__:
-            if {'message', 'httpcode', 'detail'}.issubset(set(errmsg.__dict__['anchore_error_json'])):
-                ret.update(errmsg.__dict__['anchore_error_json'])
+            # Try to load it as json
+            try:
+                err_json = json.loads(errmsg.__dict__.get('anchore_error_json', None))
```

Comment:
I usually prefer not to define `None` in a `.get()` because that is the default, so it reads redundant. I know other folks prefer it because they believe this is more explicit. Just want to emphasize that it isn't necessary to define `None` as the fallback.